ci/playbooks/multinode-customizations.yml

```diff
@@ -100,6 +100,15 @@
                 replace: >-
                   {{ _crc_default_net_ip | ansible.utils.ipaddr('address') }}
 
+            - name: Configure dnsmasq listen-address to listen on both br-ex and ci-private-network
+              become: true
+              vars:
+                _original_listen_address: "listen-address={{ hostvars['crc'].ansible_host }}"
+              ansible.builtin.replace:
```

Comment:
iirc, we could just use `ansible.builtin.lineinfile` and add the new listen-address - dnsmasq should listen on both. Might be even safer?

Reply:
So in the end have two `listen-address=` lines defined? I can quickly test that

Reply:
that's what we did for the reproducer a while ago, and it seems to work https://github.com/openstack-k8s-operators/ci-framework/blob/a1e6798dc98caae36b1c2881b77d84d20aebe965/roles/reproducer/tasks/configure_crc.yml#L87

Reply:
yup, imho that's probably better and safer than editing the default interface.

Reply:
Sorry guys, could you expand on this?

The link from @cescgina seems to match what I'm doing here. As an example we search for `listen-address=192.168.25.178` and replace it with `listen-address=192.168.25.178,192.168.122.10`

Reply:
thanks for the clarification on Slack, I confirmed lineinfile works so if your happy to go that direction I'll update this patch